src/commands/PublishCommand.js

```diff
@@ -12,11 +12,69 @@ import chalk from "chalk";
 import path from "path";
 import { EOL } from "os";
 
+const SUPPORTED_OPTS = {
```

Comment:
We should probably alphabetize the keys of this options object, just for ease of maintenance. (I think yargs alphabetizes them when listing `--help`?)